Cargo.toml

```diff
@@ -1,19 +1,29 @@
 [package]
-edition = "2021"
 name = "chainsafe_hashtree"
 version = "0.0.0"
+edition = "2021"
+authors = [
+    # TODO: @wemeetagain add your email address here
+    "Cayman Navan",
```

Comment:
do we need authors here? this crate won't ever get published, right?

Reply:
Hopefully it will 😊. Should speed up hashing in lodestar a lot

Reply:
Oh snap, good call. That should go in the package.json. You are right, the rust crate doesn't get published....